src/Bridge/Doctrine/Purger/OrmPurger.php

```diff
@@ -79,9 +79,12 @@ public function create(PurgeMode $mode, PurgerInterface $purger = null): PurgerI
 
     /**
      * @inheritdoc
+     * @throws \Doctrine\DBAL\DBALException
      */
     public function purge()
     {
+        $this->manager->getConnection()->prepare('SET FOREIGN_KEY_CHECKS = 0;')->execute();
```

Comment:
Is this necessarily? This happen only because the cascade delete or so are not set properly. Using this is a shortcut I'm using as well sometimes but I'm not sure if it should be done in the core